test/position_test.dart

```diff
@@ -129,6 +129,90 @@ void main() {
         expect(sans.item2,
             equals(['e4', 'e5', 'Bc4', 'Nc6', 'Qf3', 'd6', 'Qxf7#']));
       });
```

Comment:
On top of the tests you already wrote, I'd like to have all the tests of the chessops package as well:

https://github.com/niklasf/chessops/blob/f287455e1cc83925c9f759f3c8829e2ca89d243f/src/san.test.ts#L42-L202